addons/website/static/src/snippets/s_popup/001.scss

```diff
@@ -48,7 +48,7 @@
     .s_popup_top,
     .s_popup_bottom {
         .modal-dialog {
-            margin-right: 0;
+            margin: 0;
```

Comment:
Hello @rdeodoo ,

Thanks for the complete review! I just pushed an update to fix buttons hiding cookiebar (e.g. Livechat button).

As for your other comments, here is what I found:
- A .5rem margin is added to modals https://github.com/odoo/odoo/blob/eddc820417d97b4100a9fb2f9f8934a84bb2417c/addons/web/static/lib/bootstrap/scss/_modal.scss#L39
- The minimum height is set to 100% https://github.com/odoo/odoo/blob/eddc820417d97b4100a9fb2f9f8934a84bb2417c/addons/website/static/src/snippets/s_popup/001.scss#L12
- `overflow-y` is set to `auto` for modals https://github.com/odoo/odoo/blob/eddc820417d97b4100a9fb2f9f8934a84bb2417c/addons/web/static/lib/bootstrap/scss/_modal.scss#L13

This combined causes a scrollbar to appear.

As Nicolas mentioned in one of his previous posts (https://github.com/odoo/odoo/pull/88905#discussion_r851321447), another solution could have been to change `min-height` to remove the remaining margins but this blocks navigation through the site when the cookies bar is visible (which is not the case in "desktop" navigation).


What do you think of the fix as it stands? :slightly_smiling_face: 
Thanks!

Reply:
Note that I just saw https://github.com/odoo/odoo/pull/88905#discussion_r851321447 after reading your comment here.
Well, that was indeed the way to go? What was the exact issue?

Reply:
I understand what you mean and its true that changing popups like this into stable is maybe not a good idea.
(_As an information, i put a comparison with/without fix for a popup in mobile view that I explained in my comment a bit lower_)

To only fix the cookiebar without affecting the popup, margins can be removed from this place as proposed in my initial PR: https://github.com/odoo/odoo/blob/fda26e943a4cc37b3c97c541ce66204081dd8f40/addons/website/static/src/scss/website.scss#L1440-L1445 .

This won't impact the popup. This would be fine for you, wouldn't it?

---

`margin-right: 0` (without fix)
![popupwithoutfix](https://user-images.githubusercontent.com/91664927/168626029-e47327e9-7625-4f15-ad5e-d87aedbea629.png)

`margin: 0`
![popupmargin0](https://user-images.githubusercontent.com/91664927/168626023-b0f7b70a-667c-444c-be33-dc13c16cda8b.png)

Before the fix, the popup is shifted to the right due to the left margin but with the fix it seems to me that it is now centered. Just for information, can you explain why you say that it would now be aligned to the left? Thx :slightly_smiling_face:

Reply:
Before your fix:
![image](https://user-images.githubusercontent.com/30048408/168631801-09014595-47df-445e-bdc8-23bbe23b12b0.png)

After your fix:
![image](https://user-images.githubusercontent.com/30048408/168631759-eacc09da-2936-403b-9897-ec744ae4a8c7.png)

Not really the end of the world in itself as I said, but better to avoid it if we can :)

Reply:
> To only fix the cookiebar without affecting the popup, margins can be removed from this place as proposed in my initial PR:
Lines 1440 to 1445 in [fda26e9](https://github.com/odoo/odoo/commit/fda26e943a4cc37b3c97c541ce66204081dd8f40)
>
> This won't impact the popup. This would be fine for you, wouldn't it?

Well, why would we fix only the cookies bar and not all the popups?
See https://watch.screencastify.com/v/EzNjZHnqZFCTphXc9q4H

The fix I provided seems to be:
- Ideal as no template change, scss only, no -u needed
- Fixes the 2 problems at the root cause level
- Fixed the popups in a generic way without the need of exception of other patches

About the fact that the popup is not exactly centered in small resolution, I would not consider it in 14.0, let's stick to the 2 real "visually blocking" issues. That one always existed and is not worth introducing code in 14.0 for that.

Let me know if you find something not working with the current PR state.

Reply:
> Well, why would we fix only the cookies bar and not all the popups? See https://watch.screencastify.com/v/EzNjZHnqZFCTphXc9q4H

I was talking about the fact that the "discrete" cookiebar doesn't fill the entire width of a mobile screen. But, like non-centered popups, it's always been there and it might not be worth adding in 14.0

> Let me know if you find something not working with the current PR state.

I took a look again and it looks good to me :slightly_smiling_face: 


So we let the FW's go and stop at master (non-included )is that right? And do an IMP afterwards

Anyway, thanks again for your detailed reviews